app/models/article.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: articles
+#
+#  id           :bigint           not null, primary key
+#  for_banks    :boolean
+#  for_partners :boolean
+#  question     :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+class Article < ApplicationRecord
+  has_rich_text :content
+  validates :content, presence: true
+  validates :question, presence: true
+  validates :for_banks, acceptance: { message: "and for partners can't both be unchecked" }, unless: :for_partners
+  validates :for_partners, acceptance: { message: "and for banks can't both be unchecked" }, unless: :for_banks
```

Comment:
Isn't this really just one boolean field? It's either a bank question or a partner question - there's no such thing as either both or neither, correct?

Reply:
FAQ articles can apply to banks, partners or both. They have to apply to at least one though.